charts/sigstore-prober/values.yaml

```diff
@@ -0,0 +1,17 @@
+namespace:
+  create: true
+  name: sigstore-prober
+spec:
+  replicaCount: 1
+  serviceAccountName: default
```

Comment:
Question: aren't you missing the creation of the serviceAccount ?

Reply:
Since the service account requires workload identity enabled, I thought it might make more sense for folks to create it separately. But I can add it in if that would be useful!

Reply:
It was just to be consistent with the rest of the charts, as we also defined the SA. https://github.com/sigstore/helm-charts/blob/main/charts/ctlog/templates/ctlog-serviceaccount.yaml

Reply:
Thanks, added it in!